packages/docusaurus-remark-plugin-npm2yarn/src/index.ts

```diff
@@ -6,45 +6,67 @@
  */
 
 import visit from 'unist-util-visit';
-// @ts-expect-error: this package provides CJS
 import npmToYarn from 'npm-to-yarn';
 import type {Code, Content, Literal} from 'mdast';
 import type {Plugin} from 'unified';
 import type {Node, Parent} from 'unist';
 
+type CustomConverter = [name: string, cb: (npmCode: string) => string];
+
 type PluginOptions = {
   sync?: boolean;
+  converters?: (CustomConverter | 'yarn' | 'pnpm')[];
```

Comment:
i'm not 100% sure about this name and if we should allow to define custom converters :)

Reply:
thanks, that looks like a useful feature for userland flexibility 👍 

Maybe add a mention in our docs here?
https://docusaurus.io/docs/markdown-features/code-blocks#npm2yarn-remark-plugin

Maybe it's time to change the name of this plugin 🙈

Reply:
do you think that something like this is sufficient?

https://deploy-preview-8690--docusaurus-2.netlify.app/docs/markdown-features/code-blocks#npm2yarn-remark-plugin

Reply:
Maybe on this page, it's enough, as I don't think custom converters will be widely used, but it would be helpful if at least one custom converters example could be found.

Maybe add some extra details in the plugin readme instead of the docs? We have more room to give details on types and other things here.

Reply:
added